lib/index.ts

```diff
@@ -113,3 +117,36 @@ function substituteEnvVarValues(config): void {
     }
   });
 }
+
+function getParseEnvValues(configOptions: Options): boolean {
+  if (configOptions.parseEnvValues !== undefined) {
+    if (typeof configOptions.parseEnvValues !== 'boolean') {
+      throw new Error('options.parseEnvValues must be a boolean');
+    }
+
+    return configOptions.parseEnvValues;
+  }
+
+  const envVarVal = process.env['CONFIG_PARSE_ENV_VALUES'];
```

Comment:
I'm assuming that this env var isn't prefixed with `SNYK_` so that it doesn't get matched and included into nconf [above](https://github.com/snyk/config/pull/52/files#diff-d5a4b3a0309f2337144861084c1014523cced561eb722cb9684c63f5e41e0567R49)

If I'm right this is a somewhat obscure behaviour, would be good to leave a little note to help people in the future.

Reply:
This was done to align with the existing pattern applied in this repository. 
The same pattern is used for the `CONFIG_SECRET_FILE` environment variable, which is used to configure a file path for the secret configurations file.